### DISCUSSION TITLE
Can't find qsvp in qsv-0.137.0-x86_64-unknown-linux-gnu.zip

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2233] [Q&A] [answered]
In the releases for `qsv-0.137.0-x86_64-unknown-linux-gnu.zip` I don't see a portable binary `qsvp`.

**Comment by jqnatividad** [comment, marked as answer]
That's interesting.  That platform is in the portable publishing workflow.

https://github.com/jqnatividad/qsv/blob/master/.github/workflows/publish-portable.yml#L34-L43

Running it again...
